### PR TITLE
Add Panda-API challenge

### PR DESCRIPTION
**Pull request by cmedinilla**
https://intersysconsulting.sharepoint.com/mexico/Talent%20land%20Code%20tasks/Shared%20Documents/Irving%20Gomez%20-%20Talent%20Land.pdf

### CODE REVIEW

File: challenges/panda-api.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# PandaScore API
```
Comment (metalvegetarianoprogresivo) [inline challenges/panda-api.md]: This name is no good. The objective of this challenge is to create a web app to consume data from eSports sites. Panda Score is just the suggested API, there are others. The name should be something like *Challenge: eSport Analysis & Statistics App*.

File: challenges/panda-api.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Problema
+
+Como una compañía de Big Data, queremos crear una aplicación para nuestros clientes que quieren tener lo ultimo de información aceca de eSports, el mejor jugador, los torneos y ligas disponibles en el pasado y futuro. De esta manera nuestros clientes puedes tomar decisiones acerca de que equipos o jugadores deben invertir.
```
Comment (metalvegetarianoprogresivo) [inline challenges/panda-api.md]: This makes no sense in Spanish. Maybe something like:
```suggestion
Imginemos el siguiente escenario: somos una empresa dedicada a Big Data, por lo que queremos crear una aplicación para nuestros clientes que quieren tener lo ultimo de información aceca de eSports, el mejor jugador, los torneos y ligas disponibles en el pasado y futuro. De esta manera, nuestros clientes puedan tomar decisiones acerca de que equipos o jugadores en los que les puede convenir invertir.
```

File: challenges/panda-api.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Solución
+
+Crear una aplicación que obtenga datos de cualquier liga, jugador y torneo que nuestro cliente este interesado usando la API de Panda Score.
```
Comment (metalvegetarianoprogresivo) [inline challenges/panda-api.md]: Some changes that in Spanish do less sense:
```suggestion
Crear una aplicación que obtenga datos de cualquier liga, jugador y torneo de algun juego competitivo en el ambiente de los eSports, para que nuestro cliente este informado respecto a la información que este interesado. Para obtener la información recomendamso utilizar la API de Panda Score.
```

File: challenges/panda-api.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+[https://developers.pandascore.co/](https://developers.pandascore.co/)
+
+## Criterios de aceptación
+
+- Compatible con los browsers mas populares (desktop & mobile)
+
+- Desplegar la información en el orden que el usuario requiera.
+
+- La app debe contener la información de ligas, torneos y jugadores que estas mejor posicionados usando el API.
```
Comment (metalvegetarianoprogresivo) [inline challenges/panda-api.md]:
```suggestion
- La app debe contener la información de ligas, torneos y jugadores que estas mejor posicionados usando un API.
```

File: challenges/panda-api.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+- Desplegar gráficas e información para cada jugador.
+
+- Los datos obtenidos desde el API debe ser procesada y presentada en al APP.
```
Comment (metalvegetarianoprogresivo) [inline challenges/panda-api.md]:
```suggestion
- Los datos obtenidos desde el API deben de ser procesados y presentados en al app.
```

**Review by metalvegetarianoprogresivo — CHANGES_REQUESTED** [review]
Suggested some changes.

**Comment by cmedinilla** [comment]
♻️ @metalvegetarianoprogresivo

File: .github/issue_template.md [hunk 6]
```diff
@@ -3,6 +3,7 @@
 - [ ] Ayúdame a encontrar mi cel
 - [ ] Unicorneando las PR
 - [ ] Bot Food
+- [ ] eSport Analysis & Statistics App
```
Comment (metalvegetarianoprogresivo) [inline .github/issue_template.md]:
```suggestion
- [ ] eSports Analysis & Statistics App
```

File: README.md [hunk 7]
```diff
@@ -16,5 +16,6 @@
 - [Ayúdame a encontrar mi cel](challenges/find-me.md)
 - [Unicorneando las PR](challenges/unicorn-pr.md)
 - [Bot Food](challenges/bot-food.md)
+- [eSport Analysis & Statistics App](challenges/panda-api.md)
```
Comment (metalvegetarianoprogresivo) [inline README.md]:
```suggestion
- [eSports Analysis & Statistics App](challenges/esports_app.md)
```

File: challenges/panda-api.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
+# eSport Analysis & Statistics App
```
Comment (metalvegetarianoprogresivo) [inline challenges/panda-api.md]:
```suggestion
# eSports Analysis & Statistics App
```

File: challenges/panda-api.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+## Problema
+
+Imaginemos el siguiente escenario: somos una empresa dedicada a Big Data, por lo que queremos crear una aplicación para nuestros clientes que quieren tener lo ultimo de información aceca de eSports, el mejor jugador, los torneos y ligas disponibles en el pasado y futuro. De esta manera, nuestros clientes puedan tomar decisiones acerca de que equipos o jugadores en los que les puede convenir invertir.
+
+## Solución
+
+Crear una aplicación que obtenga datos de cualquier liga, jugador y torneo de algun juego competitivo en el ambiente de los eSports, para que nuestro cliente este informado respecto a la información que este interesado. Para obtener la información recomendamos utilizar la API de Panda Score.
```
Comment (metalvegetarianoprogresivo) [inline challenges/panda-api.md]:
```suggestion
Crear una aplicación que obtenga datos de cualquier liga, jugador y torneo de algun juego competitivo en el ambiente de los eSports, para que nuestro cliente este informado respecto a la información que este interesado. Para obtener la información necesaria para realizar este ejercicio, te sugerimos utilizar la API de Panda Score.
```

**Review by metalvegetarianoprogresivo — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (cmedinilla) [reply README.md]: We are not using underscore for the other files

**Comment by cmedinilla** [comment]
♻️ @metalvegetarianoprogresivo

File: README.md (hunk 7 above)
Comment (cmedinilla) [reply README.md]: `- [eSports Analysis & Statistics App](challenges/esports-app.md)`

File: challenges/esports-app.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+## Problema
+
+Imaginemos el siguiente escenario: somos una empresa dedicada a Big Data, por lo que queremos crear una aplicación para nuestros clientes que quieren tener lo ultimo de información aceca de eSports, el mejor jugador, los torneos y ligas disponibles en el pasado y futuro. De esta manera, nuestros clientes puedan tomar decisiones acerca de que equipos o jugadores en los que les puede convenir invertir.
+
+## Solución
+
+Crear una aplicación que obtenga datos de cualquier liga, jugador y torneo de algún juego competitivo en el ambiente de los eSports, para que nuestro cliente este informado respecto a la información que este interesado. Para obtener la información necesaria para realizar este ejercicio, te sugerimos utilizar la API de Panda Score.
+
+[https://developers.pandascore.co/](https://developers.pandascore.co/)
```
Comment (metalvegetarianoprogresivo) [inline challenges/esports-app.md]:
```suggestion
- [PandaScore](https://developers.pandascore.co/)
- [Abios Esports API](https://abiosgaming.com/esports-data/api/)
```

File: challenges/esports-app.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
+# eSports Analysis & Statistics App
+
+## Problema
+
+Imaginemos el siguiente escenario: somos una empresa dedicada a Big Data, por lo que queremos crear una aplicación para nuestros clientes que quieren tener lo ultimo de información aceca de eSports, el mejor jugador, los torneos y ligas disponibles en el pasado y futuro. De esta manera, nuestros clientes puedan tomar decisiones acerca de que equipos o jugadores en los que les puede convenir invertir.
+
+## Solución
+
+Crear una aplicación que obtenga datos de cualquier liga, jugador y torneo de algún juego competitivo en el ambiente de los eSports, para que nuestro cliente este informado respecto a la información que este interesado. Para obtener la información necesaria para realizar este ejercicio, te sugerimos utilizar la API de Panda Score.
```
Comment (metalvegetarianoprogresivo) [inline challenges/esports-app.md]: Encontre otra API que puede usar como referencia.
```suggestion
Crear una aplicación que obtenga datos de cualquier liga, jugador y torneo de algún juego competitivo en el ambiente de los eSports, para que nuestro cliente este informado respecto a la información que este interesado. Para obtener la información necesaria para realizar este ejercicio, te sugerimos utilizar alguna de estas APIs:
```

**Review by metalvegetarianoprogresivo — CHANGES_REQUESTED** [review]
Add the changes that I suggested and it's good to go.

**Review by metalvegetarianoprogresivo — APPROVED** [review]

**Comment by metalvegetarianoprogresivo** [comment]
Good to merge!

**Pull request merged**